src/templates/template.cpp

```diff
@@ -130,16 +136,23 @@ void TemplateTransform::load(QXmlStreamReader& xml)
 	template_y = coord.nativeY();
 	template_scale_x = element.attribute<double>(QLatin1String("scale_x"));
 	template_scale_y = element.attribute<double>(QLatin1String("scale_y"));
+	if (element.hasAttribute(QLatin1String("shear")))
+		template_shear = element.attribute<double>(QLatin1String("shear"));
+	else
+		template_shear = 0.0;
```

Comment:
`element.attribute<double>(QLatin1String("shear"))` will return 0 anyway if the attribute doesn't exist. So no `if else` needed here.

Reply:
Simplified as indicated.